packages/augur-ui/src/modules/universe/actions/load-forking-info.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  getForkingMarket,
+  getForkEndTime,
+  getForkReputationGoal,
+  isFinalized,
+  getWinningChildUniverse,
+  getDisputeThresholdForFork,
+} from 'modules/contracts/actions/contractCalls';
+import { ThunkDispatch } from 'redux-thunk';
+import { AppState } from 'store';
+import { Action } from 'redux';
+import { updateUniverse } from 'modules/universe/actions/update-universe';
+import { ForkingInfo } from 'modules/types';
+
+export function loadUniverseForkingInfo(
+  incomingUniverse: string,
+  forkingMarketId?: string
+) {
+  return async (
+    dispatch: ThunkDispatch<void, any, Action>,
+    getState: () => AppState
+  ) => {
+    const { universe } = getState();
+    if (universe && universe.id && universe.id !== incomingUniverse) return;
+    const forkingMarket = forkingMarketId || (await getForkingMarket());
+    const isForking =
+      forkingMarket !== '0x0000000000000000000000000000000000000000';
```

Comment:
Isn't there a constant for NULL_ADDRESS that can be used here?

Reply:
good one I'll change that